hazelcast/src/main/java/com/hazelcast/client/impl/operations/ClientDisconnectionOperation.java

```diff
@@ -30,7 +32,7 @@
 import java.util.Collection;
 import java.util.Set;
 
-public class ClientDisconnectionOperation extends Operation implements UrgentSystemOperation {
+public class ClientDisconnectionOperation extends Operation implements UrgentSystemOperation, IdentifiedDataSerializable {
```

Comment:
In general yes, there were just 4 client operations though. I changed it as suggested.